cli/azd/cmd/monitor.go

```diff
@@ -57,7 +58,7 @@ func (m *monitorAction) SetupFlags(
 
 func (m *monitorAction) Run(ctx context.Context, _ *cobra.Command, args []string, azdCtx *environment.AzdContext) error {
 	azCli := commands.GetAzCliFromContext(ctx)
-	askOne := makeAskOne(m.rootOptions.NoPrompt)
+	console := input.NewAskerConsole(!m.rootOptions.NoPrompt)
```

Comment:
consider keeping the same input-arg logic for `NewAskerConsole` so we don't need to add the negation all the times (`!`)

Reply:
I like Wallace's direction of `Interactive` over `NoPrompt`. I think we could avoid the negation for the current change, and as a separate change, refactor all usages of `NoPrompt` (including in `rootOptions`) -> `Interactive`. That way we avoid the bool conversions.